### PR TITLE
length.md added in the java folder 

### LINKED ISSUE #1451: [Term Entry] Java Array: `.length` property
### Reason/inspiration (optional)

Add a new term entry under the [Java: Array](https://www.codecademy.com/resources/docs/java/array-list) concept. This entry should include:

- An explanation of the property.
- A "Syntax" section with a ```pseudo code snippet showing the general syntax.
- An "Example" section with a ```java code snippet showing how it works.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by dakshdeepHERE**
### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

Fixes #1451

### CODE REVIEW

File: content/java/concepts/array-list/terms/length/length.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+In Java, the `.length()` method is a member of the `String` class that returns the length of a `string`, in terms of number of characters.
+
+## Syntax
+
+```pseudo
+int length = str.length();
+```
+
+Here, `str` is a string variable or a string literal, and length is an integer variable that will hold the length of the string.
```
Comment (SSwiniarski) [inline content/java/concepts/array-list/terms/length/length.md:24]:
```suggestion
Here, `str` is a string variable or a string literal, and `length` is an integer variable that will hold the length of the string.
```

File: content/java/concepts/array-list/terms/length/length.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Example
```
Comment (SSwiniarski) [inline content/java/concepts/array-list/terms/length/length.md:26]: We'd like a sentence of description preceding the example describing what it's doing.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@dakshdeepHERE, I did a first review and left a couple of comments.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@dakshdeepHERE, I looked a little closer and discovered a few things that change what needs to happen to this entry:

- This is for an `ArrayList` not a `String` so the wording and examples need to change.
- The actual issue was misstated, as an `ArrayList` doesn't have a `.legnth()` property, it has a `.size()` property, so the file and title need to be renamed.

If you can revise this along these lines, I can come back and do a line by line review.

**Comment by dakshdeepHERE** [comment]
@SSwiniarski yes I also didnt notice this when making the doc as you can see I wrote string there too size() is used for arraylist and length() is used for strings Ok I can change this for arraylist that is size().

**Comment by SSwiniarski** [comment]
@dakshdeepHERE I see you have a separate PR for ArrayList .size, were you going to change this to Array .legnth or did you just want to close it?

**Comment by dakshdeepHERE** [comment]
@SSwiniarski just close it because I made a diff pr with .size() already :)

**Pull request closed without merging**